spec/index.bs

```diff
@@ -1013,7 +1183,9 @@ To get a {{GPUAdapter}}, use {{GPU/requestAdapter()}}.
 interface GPUAdapter {
     readonly attribute DOMString name;
     readonly attribute FrozenArray<GPUExtensionName> extensions;
-    //readonly attribute GPULimits limits; Don't expose higher limits for now.
+
+    boolean supportsLimit(GPULimitName name, double value);
+    double getLimit(GPULimitName name);
```

Comment:
what's the motivation for having both `supportsLimit` and `getLimit`?

Reply:
Sometimes the app just needs to know which of two modes to operate in (say, 4 bind groups or 8 bind groups) and doesn't care if it's higher. `supportsLimit('maxBindGroups', 8)` theoretically exposes fewer bits of entropy than `getLimit('maxBindGroups')`.